src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -50,25 +50,16 @@ public bool SupportsThrowExpression(ParseOptions options)
         public SyntaxToken ParseToken(string text)
             => SyntaxFactory.ParseToken(text);
 
-        public bool IsAwaitKeyword(SyntaxToken token)
-        {
-            return token.IsKind(SyntaxKind.AwaitKeyword);
-        }
+        public bool IsAwaitKeyword(SyntaxToken token) => token.IsKind(SyntaxKind.AwaitKeyword);
```

Comment:
Note: we have an ISyntaxKindsService.  These should move there.  I realize that you aren't changing anything.  But my view on code is: if you're touching it, then push it all the way to the correct state :D

Reply:
It seems that a lot of things would need to move, better do it within a quick standalone PR. I'll make a note to fix that.

Reply:
WFM.  Thanks!

Reply:
Ok, got to look at that and am not sure what exactly did you mean. Did you mean that we should create fields for AwaitKeyword in ISyntaxKindsService and then lift this method to the abstract base?